networkoperator/v1/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package v1 contains API Schema definitions for the network v1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=network.operator.openshift.io
```

Comment:
set optional as default here for the whole package. Then you can drop all the optional markers in the types. It's done like that for most other packages in this repo.

Reply:
oh cool,! didn't know this existed. will change this and drop all optional fields.